backend/package.json

```diff
@@ -14,11 +14,18 @@
   "dependencies": {
     "@filecoin-station/deal-observer-db": "^1.0.0",
     "@influxdata/influxdb-client": "^1.35.0",
+    "@ipld/dag-cbor": "^9.2.2",
+    "@ipld/dag-json": "^10.2.3",
+    "@ipld/schema": "^6.0.5",
```

Comment:
Do we still need `@ipld/schema` now that you are using typebox? If not, then please remove this unused dependency.

Reply:
@NikolasHaimerl PTAL ☝🏻